src/js/stores/AppFormStore.js

```diff
@@ -161,11 +161,11 @@ const responseAttributePathToFieldIdMap = {
     "healthChecks/{INDEX}/maxConsecutiveFailures",
   "/instances": "instances",
   "/mem": "mem",
-  "portDefinitions": "portDefinitions",
-  "portDefinitions[{INDEX}]/name": "portDefinitions/{INDEX}/name",
-  "portDefinitions[{INDEX}]/port": "portDefinitions/{INDEX}/port",
-  "portDefinitions[{INDEX}]/protocol": "portDefinitions/{INDEX}/protocol",
-  "container/docker/portMappings": "portDefinitions",
+  "/portDefinitions": "portDefinitions",
+  "/portDefinitions[{INDEX}]/name": "portDefinitions/{INDEX}/name",
```

Comment:
Is this confirmed with the slashes?

It was by intetion without the slashes, but could be changed in the meantime. Just curious..